### ISSUE TITLE
Use `go install` instead of `go get`

### ISSUE THREAD

**Issue by dipeshdulal** [WesionaryTEAM/go_clean_architecture#63]
`go get` no longer supports installing binaries (in the latest version of go 1.18). The work has been moved to the `install` command. The docker installation command needs to be updated using `install` instead of `go get`

```Dockerfile
RUN go install github.com/rubenv/sql-migrate/...@latest
RUN go install github.com/go-delve/delve/cmd/dlv@master
```

**Comment by dipeshdulal** [comment]
Merging #61 will close this issue.

**Comment by paudelgaurav** [comment]
This issue is already resolved by #78 so I am closing it.
